tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "sourceMap": false,
     "declaration": true,
     "removeComments": false,
-    "noImplicitAny": false,
```

Comment:
`noImplicitAny` is helpful to ensure types in the method signature. If we need to be lenient, we could use `any` as the type.